product_docs/docs/eprs/7/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -93,9 +93,9 @@ $ rm server.csr
 **Step 5:** Move or copy the certificate and private key files to the Postgres database server data directory, `POSTGRES_INSTALL_HOME/data`.
 
 ```text
-$ mv root.crt /opt/PostgresPlus/9.4AS/data
-$ mv server.crt /opt/PostgresPlus/9.4AS/data
-$ mv server.key /opt/PostgresPlus/9.4AS/data
+$ mv root.crt /opt/PostgresPlus/as14/data
```

Comment:
The new path is "/var/lib/edb/" while older path was "/opt/PostgresPlus/" . Please update the paths to be /var/lib/edb/as14/data